libpermian/webui/static/style.css

```diff
@@ -113,7 +113,8 @@ h1 {
 
 .crc_logs ul {
     max-height: 2.2em;
-    overflow-y: hidden;
+    max-width: 120px;
```

Comment:
I find this problematic as I've just tried this change with the demo command on 2560x1440 resolution with sidebar on in Firefox and when I expanded logs of some crc containing long log name the page totally re-aligned (due to the floating test runs) visually losing the crc I interacted with as well as the testruns.

Reply:
It would make more sense to me to not change the width but rather show/hide the horizontal scrollbar when the logs are collapsed/expanded.

Reply:
Good catch, thanks

Reply:
Looks good now, thank you.